packages/forma-36-codemod/utils/changeComponentName.js

```diff
@@ -0,0 +1,15 @@
+module.exports.changeComponentName = function changeProperties(
+  j,
```

Comment:
Same suggestion! It's hard to know what `j` is
I think a descriptive name would help us to maintain this code